PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -76,23 +76,31 @@
         'keep CTPPSLocalTrackLites_ctppsLocalTrackLiteProducer_*_*'
     )
 )
-MicroEventContentMC = cms.PSet(
-    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
-)
-MicroEventContentMC.outputCommands += [
-        #'keep *_slimmedGenJets*_*_*',
-        'keep *_slimmedGenJets_*_*',
-        'keep *_slimmedGenJetsAK8_*_*',
+
+MicroEventContentGEN = cms.PSet(
+    outputCommands = cms.untracked.vstring(
         'keep patPackedGenParticles_packedGenParticles_*_*',
         'keep recoGenParticles_prunedGenParticles_*_*',
         'keep LHEEventProduct_*_*_*',
-        'keep PileupSummaryInfos_slimmedAddPileupInfo_*_*',
         'keep GenFilterInfo_*_*_*',
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep *_slimmedGenJets_*_*',
         # RUN
         'keep LHERunInfoProduct_*_*_*',
         'keep GenRunInfoProduct_*_*_*',
-        'keep L1GtTriggerMenuLite_l1GtTriggerMenuLite__*',
-]
+    )
+)
+
+MicroEventContentMC = cms.PSet(
+    outputCommands = cms.untracked.vstring(MicroEventContent.outputCommands)
+)
+MicroEventContentMC.outputCommands += MicroEventContentGEN.outputCommands
+MicroEventContentMC.outputCommands += [
+                                        'keep *_slimmedGenJetsAK8_*_*',
```

Comment:
wouldn't it make sense, at least from the point of view of consistency, to keep also the `slimmedGenJetsAK8` ?